Digital Clock/Button192x50.qml

```diff
@@ -0,0 +1,55 @@
+import QtQuick
```

Comment:
Tutti questi Button diversi mi suonano strani. Già il fatto che nel nome ci sia finita la dimensione mi suggerisce che in realtà siano la stessa cosa solo con piccole modifiche. La grandezza di un pulsante non dovrebbe influenzare la rappresentazione grafica. È possibile diminuire il numero?